src/model/signatureMatrix.js

```diff
@@ -4,9 +4,11 @@ import { signatures } from "./signatures";
 const COLUMNS = 30;
 const ROWS = 20;
 
+// 
```

Comment:
I've noticed a lot of "floating" comments. Can you ensure to clean these up being commiting?